xnat_tools/bids_utils.py

```diff
@@ -65,38 +67,50 @@ def prepare_bids_prefixes(project, subject, session):
 
     return pi_prefix, study_prefix, subject_prefix, session_prefix
 
-def prepare_bids_output_path(bids_root_dir, pi_prefix, study_prefix, subject_prefix, session_prefix):
+def prepare_export_output_path(bids_root_dir, pi_prefix, study_prefix, subject_prefix, session_prefix, overwrite=False):
 
-    bids_study_dir = os.path.join(bids_root_dir, pi_prefix, study_prefix)
-    bids_subject_dir = os.path.join(bids_study_dir, "xnat-export", subject_prefix)
-    bids_session_dir = os.path.join(bids_subject_dir, session_prefix)
+    study_dir = os.path.join(bids_root_dir, pi_prefix, study_prefix)
+    subject_dir = os.path.join(study_dir, "xnat-export", subject_prefix)
+    session_dir = os.path.join(subject_dir, session_prefix)
 
     # Set up working directory
-    if not os.access(bids_session_dir, os.R_OK):
-        _logger.info('Making output BIDS Session directory %s' % bids_study_dir)
-        os.makedirs(bids_session_dir)
+    if overwrite and os.path.exists(session_dir):
+        _logger.info('Removing existing xnat-export session directory %s' % session_dir)
+        shutil.rmtree(session_dir, ignore_errors=True)
+
+    if not os.access(session_dir, os.R_OK):
+        _logger.info('Making output xnat-export session directory %s' % session_dir)
+        os.makedirs(session_dir)
```

Comment:
That was a line that has always been there, which came form code I started with. Maybe it was because it was run from a container, and it was checking if it was local/remote call

But I agree the logic would be better with 
```
if not os.path.isdir(session_dir):
  os.makedirs(session_dir)
```

I don't think we need to worry about edge cases.